cdc/sinkv2/ddlsink/mq/mq_ddl_sink.go

```diff
@@ -175,4 +181,12 @@ func (k *ddlSink) Close() {
 	if k.producer != nil {
 		k.producer.Close()
 	}
+	if k.admin != nil {
+		if err := k.admin.Close(); err != nil {
```

Comment:
What would happen if we don't throw this error to caller? Will the error neglect cause problems like goroutine leak or something else?

Reply:
It's unnecessary to report errors on closing to the caller, just print the log is enough.

No problems like goroutine leak.